bowtie/tests/test_integration.py

```diff
@@ -2247,28 +2247,20 @@ async def test_info_links():
     )
     assert stdout == dedent(
         f"""\
-        name: "links"
-        language: "python"
-        version: "v1.0.0"
-        homepage: "urn:example"
-        issues: "urn:example"
-        language_version: "{platform.python_version()}"
-        os: "{platform.system()}"
-        os_version: "{platform.release()}"
-        source: "urn:example"
-        dialects: [
-          "http://json-schema.org/draft-07/schema#"
-        ]
-        links: [
-          {{
-            "description": "foo",
-            "url": "urn:example:foo"
-          }},
-          {{
-            "description": "bar",
-            "url": "urn:example:bar"
-          }}
-        ]
+        ╭──────────────────┬─────────────────────────────────────────╮
+        │ name             │ links                                   │
+        │ language         │ python                                  │
+        │ version          │ v1.0.0                                  │
+        │ homepage         │ urn:example                             │
+        │ issues           │ urn:example                             │
+        │ language version │ {platform.python_version():39} │
```

Comment:
This seems very likely to break if we change the terminal size we're simulating. Probably at least you should reference that value, I think we have a constant somewhere here in the test file.